src/types/types.ts

```diff
@@ -0,0 +1,12 @@
+import { Dispatch } from '@sumup/collector/build/types';
```

Comment:
The types should be exposed by `@sumup/collector` at the top level so that the folder structure can remain an implementation detail.

Reply:
I'm not sure if this is only in my vscode or not. but I'll have an error saying: Module '"xx/xx/.../circuit-ui/node_modules/@sumup/collector/build"' has no exported member 'Dispatch' when I import only from "@sumup/collector"

Reply:
Oh yeah, I meant that we need to add it to the exports in `@sumup/collector` first, before we can import it here. Sorry I wasn't clear enough.

Reply:
Ahhh not at all. it's me didn't get it. Thank you. let me check how to export this in @sumup/collector!!